src/utils/jwt.ts

```diff
@@ -89,7 +89,7 @@ export class JWT {
    * const token = Mux.JWT.sign('some-playback-id', { keyId: 'your key id', keySecret: 'your key secret' });
    * // Now you can use the token in a url: `https://stream.mux.com/some-playback-id.m3u8?token=${token}`
    */
-  static sign(playbackId: string, options: MuxJWTSignOptions = {}) {
```

Comment:
Rather than completely eliminating the `sign` method, can we make it a deprecated alias for signPlaybackId?